gsa/src/web/pages/login/loginform.js

```diff
@@ -93,62 +101,85 @@ class LoginForm extends React.Component {
   }
 
   render() {
-    const {error} = this.props;
+    const {
+      error,
+      showGuestLogin = false,
+      showLogin = false,
```

Comment:
If this is false by default, shouldn't NOT setting it to true explicitly on the LoginPage not show the LoginForm? I.e. when rendering ```<LoginForm/>``` we would need the prop ```showLogin={true}```

Reply:
Nevermind. It's done in the next commit...

Reply:
It's updated to showLogin=true in a later commit ;-)